doc/en/example/parametrize.rst

```diff
@@ -506,8 +502,12 @@ Running it results in some skips if we don't have all the python interpreters in
 .. code-block:: pytest
 
    . $ pytest -rs -q multipython.py
-   ...........................                                          [100%]
-   27 passed in 0.12s
+   sssssssssssssssssssssssssss                                          [100%]
+   ========================= short test summary info ==========================
+   SKIPPED [9] multipython.py:29: 'python3.5' not found
+   SKIPPED [9] multipython.py:29: 'python3.6' not found
+   SKIPPED [9] multipython.py:29: 'python3.7' not found
+   27 skipped in 0.12s
```

Comment:
Hmmm, this output doesn't really show what we wanted to show there. Not sure what to do about that?

Reply:
No I think showing a few skips is fine. I wouldn't worry about it.

Reply:
we could jsut run setup-python for all pythons we need, sometime later

Reply:
We say:

> Running it results in some skips if we don't have all the python interpreters installed and otherwise runs all combinations (3 interpreters times 3 interpreters times 3 objects to serialize/deserialize):

but I guess "all skipped" is an alright demo of "some skips" :laughing: